SUMMARY.md

```diff
@@ -132,15 +132,19 @@
 * [Media API](api-reference/media-api/README.md)
   * [List and search files](api-reference/media-api/list-and-search-files.md)
   * [Get file details](api-reference/media-api/get-file-details.md)
+  * [Get file version details](api-reference/media-api/get-file-version-details.md)
+  * [Get version details](api-reference/media-api/get-versions-details.md)
```

Comment:
Should it be just `Get file versions` and corresponding change in `.md` file name and docs URL?